packages/insomnia-app/app/plugins/context/__tests__/app.test.ts

```diff
@@ -11,12 +14,18 @@ describe('init()', () => {
     expect(Object.keys(result)).toEqual(['app', '__private']);
     expect(Object.keys(result.app).sort()).toEqual([
       'alert',
+      'clipboard',
       'dialog',
       'getPath',
       'prompt',
       'showGenericModalDialog',
       'showSaveDialog',
     ]);
```

Comment:
Curious how you feel about sorting both the expected and the result keys so that the test is indifferent to order since I don’t think that’s what we’re actually trying to test here? Could also use a linting rule if we're sorting more for the developer experience (let machines do what they do best).

```suggestion
    ].sort());
```

Reply:
Heh, meet @dimitropoulos, who is also super keen on sorted properties 😝 I think we'll get there, eventually!

That sounds reasonable to me though, in this instance it would be better to use `.sort()` in code because the expect block also does a `.sort()`. Nice catch!